index.js

```diff
@@ -68,7 +68,7 @@ function buildAPI(loginOptions, html, jar) {
 
   var apiFuncNames = [
     'listen',
-    'getUserID',
+    'getUserId',
```

Comment:
whoops. sorry, when i was testing, the module didn't work because it cant load ```getUserID``` because you didn't change the filename.